### DISCUSSION TITLE
I didnt got my benefits pack

### DISCUSSION THREAD

**Discussion by RoeeANahum** [community/community#145553] [GitHub Education]
### Hi everyone,

i didnt got my benefits and my request was approved on the 17 of november what should i do

**Comment by davevad93** [comment]
Hi @RoeeANahum ,

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @RoeeANahum was approved and your benefits are now available.`

For further reference you can look at these discussions: 
#132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268, #140287

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Comment by Hiroohamada** [comment]
I haven't
![Screenshot 2024-11-26 071602](https://github.com/user-attachments/assets/4b011161-a6b1-47af-82c8-9b20e656954e)
 received my student developer pack yet. I applied it on 23rd of November.
